presto-orc/src/main/java/com/facebook/presto/orc/Stripe.java

```diff
@@ -29,13 +30,15 @@
     private final Map<Integer, ColumnEncoding> columnEncodings;
     private final List<RowGroup> rowGroups;
     private final InputStreamSources dictionaryStreamSources;
+    private final IntegerDictionaryProvider integerDictionaryProvider;
```

Comment:
I like how it's just called dictionaryProvider in the classes below, consider renaming it to dictionaryProvider.